python/setup.py

```diff
@@ -144,14 +142,10 @@ def run(self):
                      ('with-gcs', None,
                       'build the Google Cloud Storage (GCS) extension'),
                      ('with-s3', None, 'build the Amazon S3 extension'),
-                     ('with-static-parquet', None, 'link parquet statically'),
```

Comment:
Are these changes relevant to this PR?

Reply:
No, but these options are entirely unused.